rust/src/answer/readable_concept.rs

```diff
@@ -81,41 +85,52 @@ impl Node {
     }
 }
 
+const TYPE: Cow<'static, str> = Cow::Borrowed("type");
```

Comment:
Nice I like this COW approach, we will leverage it more in the future!

Reply:
Note, `Cow` is a good solution for when the repeated string is static, but we should look to do proper string interning to help with type labels and other runtime strings.